Bots/DotNet/Skills/Composer/EchoComposerSkillBot/runtime/customaction/CustomActionComponentRegistration.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Bot.Builder;
```

Comment:
The copyright header is missing here.

Reply:
Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)